plugins/millumin/export.js

```diff
@@ -0,0 +1,20 @@
+module.exports.name = `Millumin`;
+module.exports.version = `0.0.1`;
```

Comment:
This should be either `7.0.0` or the added `$schema` property should use `7.0.0` instead of `module.exports.version`